src/client/delphi_epidata.py

```diff
@@ -45,10 +45,15 @@ class Epidata:
 
     client_version = __version__
 
-    logger = get_structured_logger('delphi_epidata_client')
     debug = False # if True, prints extra logging statements
     sandbox = False # if True, will not execute any queries
 
+    @staticmethod
+    def log(evt, **kwargs):
+        kwargs['event'] = evt
+        kwargs['timestamp'] = time.strftime("%Y-%m-%d %H:%M:%S %z")
+        return sys.stderr.write(str(kwargs) + "\n")
```

Comment:
this might need a `flush()` call to keep it synced
```suggestion
        sys.stderr.write(str(kwargs) + "\n")
        sys.stderr.flush()
```

Reply:
TL;DR: we don't need the `flush()`.

```py
>>> sys.stderr
<_io.TextIOWrapper name='<stderr>' mode='w' encoding='utf-8'>
>>> sys.stderr.line_buffering
True
```

[`io.TextIOWrapper`](https://docs.python.org/3/library/io.html#io.TextIOWrapper) says this about `line_buffering=True`

> If line_buffering is True, [flush()](https://docs.python.org/3/library/io.html#io.IOBase.flush) is implied when a call to write contains a newline character or a carriage return.